package.json

```diff
@@ -38,14 +41,14 @@
   },
   "authors": [
     "Josh Perez <josh@goatslacker.com>",
-    "Jonathan Lehman <jonathan.lehman91@gmail.com>"
+    "Jonathan Lehman <jonathan.lehman91@gmail.com>",
+    "Jaroslaw Zabiello <hipertracker@gmail.com>"
```

Comment:
This space is for the project maintainers not all the [contributors](https://github.com/goatslacker/alt/graphs/contributors) although I'm not opposed to adding a link to that page or mentioning it somewhere else :)